project/Dependencies.scala

```diff
@@ -89,16 +89,18 @@ object Dependencies {
 
   val workbenchModelV  = s"0.15-${workbenchLibsHash}"
   val workbenchGoogleV = s"0.21-${workbenchLibsHash}"
-  val workbenchNotificationsV = s"0.3-${workbenchLibsHash}"
+  val workbenchNotificationsV = s"0.3-a79c7f9"
```

Comment:
Perhaps I should add this in a code comment too, but the reason I am only moving workbenchNotifications to `a79c7f9` rather than moving `workbenchLibsHash` to `a79c7f9` is that it seemed to initiate a hell-spiral of backwards incompatible Google library changes, some of which were runtime errors rather than compilation errors. For example, it broke the listBillingAccounts endpoint in a silent way. In the interest of keeping this PR simple, I opted to only update the notifications lib.

Reply:
I do think it is worth a code comment, and a ticket in the backlog to update `workbenchLibsHash` (unless one already exists).

Reply:
I created SU-278 (which I will add a description to shortly) and added a pointer to it in a comment: 422a1a4ce92ae57e2b68c4478024bacf42e7a7fb